### ISSUE TITLE
Carousel content is not located in an aria-polite zone and is not read by screen readers when element becomes active

### ISSUE THREAD

**Issue by blackholegalaxy** [primefaces/primeng#11515]
**I'm submitting a ...**
```
[x] bug report => Search github for a similar issue or PR before submitting
```

**Repro Case (Bug Reports)**
https://www.primefaces.org/primeng/carousel

Happens on official demo.

**Current behavior**
No `aria-polite` tag is there. Also please note aria-polite sections need to be defined before the content is changed to be read properly by screen readers.

**Expected behavior**
Having a `aria-polite` section somewhere in the component to ensure proper active information reading by screen readers.

**What is the motivation / use case for changing the behavior?**
Accessibility. Currently, dynamic content is NOT read by screen readers properly when they should.

**Please tell us about your environment:**
All

* **Angular version:** 13

* **PrimeNG version:** 13.4.0

* **Browser:** all

**Comment by cetincakiroglu** [comment]
Hi,

We're fully focused on Accessibility. Thanks so much for sharing these with us.
Our main goal is to consider all the components one by one and document the individual accessibility features. In doing so, it is to correct the deficiencies or faulty implementations of that component. For this reason, our team carries out a comprehensive and meticulous study. Of course, it will take some time to complete this process.
Currently, working on the components specified in your attached documentation will indeed be a temporary solution for us and we cannot guarantee that these solutions will not change. Because we examine a lot of accessibility documents and try to make the best decisions for users. At this stage, the best decision that can be made about such issues is to stop the actions to be taken on them for the time being. We will reactivate these issues after we complete our work on accessibility in Prime* component libraries. In this way, we will be able to make better decisions about issues.
Details; https://www.primefaces.org/primeng/accessibility
Thanks a lot for your understanding! 

Best Regards
